.github/workflows/ubuntu-latest-oneapi.yml

```diff
@@ -0,0 +1,45 @@
+name: Build - Ubuntu/dpcpp
+
+on:
+  pull_request:
+  merge_group:
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.ref_name }}
+  cancel-in-progress: true
+
+env:
+  # Customize the CMake build type here (Release, Debug, RelWithDebInfo, etc.)
+  BUILD_TYPE: Release
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        ONEAPI_VERSION: [
+          '2024.2.1-0-devel-ubuntu22.04',
+          'latest' # 2025.0.0-0-devel-ubuntu24.04 at the time of commit
+        ]
+
+    runs-on: ubuntu-latest
+    container: intel/oneapi-basekit:${{ matrix.ONEAPI_VERSION }}
```

Comment:
Where are these containers hosted? I thought they'd be at https://github.com/orgs/intel/packages but I don't see them there.

Reply:
They are hosted in the [default docker registry](https://hub.docker.com/r/intel/oneapi-basekit).